sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubAsyncClientIntegrationTest.java

```diff
@@ -180,15 +181,15 @@ public void sendAndReceiveEventByAzureSasCredential() {
                 "SAS was not set. Can't run test scenario.");
 
         ConnectionStringProperties properties = getConnectionStringProperties(true);
-        String fullyQualifiedNamespace = getFullyQualifiedDomainName();
+        String fullyQualifiedNamespace = properties.getEndpoint().getHost();
         String sharedAccessSignature = properties.getSharedAccessSignature();
-        String eventHubName = getEventHubName();
+        String eventHubName = properties.getEntityPath();
 
         final EventData testData = new EventData(TEST_CONTENTS.getBytes(UTF_8));
 
         EventHubProducerAsyncClient asyncProducerClient = new EventHubClientBuilder()
                 .credential(fullyQualifiedNamespace, eventHubName,
-                        new AzureSasCredential(sharedAccessSignature))
+                        new EventHubSharedKeyCredential(sharedAccessSignature))
```

Comment:
This test case is to test a SasCredential.

Reply:
The code has been rollback to the original settings and successfully passed the test on UsGov and China cloud by pipeline.